demos/unassigned_or_unused/tutorial_alignments/alignment_local_waterman_eggert.cpp

```diff
@@ -1,30 +0,0 @@
-//![main]
-#include <iostream>
-#include <seqan/align.h>
-
-using namespace seqan;
-
-int main()
-{
-//![main]
-
-//![init]
-    Align<String<char> > ali;
-    resize(rows(ali), 2);
-    assignSource(row(ali, 0), "ataagcgtctcg");
-    assignSource(row(ali, 1), "tcatagagttgc");
-//![init]
-
-//![alignment]
-    Score<int> scoring(2, -1, -2, 0);
-    LocalAlignmentEnumerator<Score<int>, Unbanded> enumerator(scoring, 5);
```

Comment:
We describe it but in another demo.
So this is obsolete and can be removed. 
I also couldn't find any source that is including this file.